css/site.css

```diff
@@ -544,10 +544,13 @@ body {
   background-color: #000000!important;
   border-color: #000000!important; /*set the color you want here*/
 }
+.tn-events-keyword-filter__keyword-input:hover + .tn-events-keyword-filter__keyword-button, .tn-events-keyword-filter__keyword-input:active:focus + .tn-events-keyword-filter__keyword-button, .tn-events-keyword-filter__keyword-input:focus + .tn-events-keyword-filter__keyword-button, .tn-events-keyword-filter__keyword-input:active + .tn-events-keyword-filter__keyword-button, .tn-events-keyword-filter__keyword-input.active + .tn-events-keyword-filter__keyword-button, .tn-events-keyword-filter__keyword-input.open + .tn-events-keyword-filter__keyword-button,
```

Comment:
Hi @ostafinskim , just checking, it looks like this change was made so that these elements had a visible and different focus state?

However it looks like some of the elements also have hover and active states defined to use the same styles. Is there a reason for that?

Reply:
This has been done because to enable visible focus state on element while tabbing, also to persist correct text color inside element.